types/k6/test/tracing.ts

```diff
@@ -0,0 +1,214 @@
+import http, {
```

Comment:
@oleiade, the test is for the experimental/tracing API, it seems the `http` import is unnecessary.

Reply:
Good catch, I guess in this case I've been failed by my linter 🥷 Fixing it 👍🏻